### PR TITLE
Refs CORE-5412

### PR DESCRIPTION
**Pull request by rozap**
* Added polygon, multipoint, and line types
* added tests
* added test fixtures
* TODO: update the dependencies file and remove the snapshot reference

### CODE REVIEW

File: soql-server-pg/src/test/resources/fixtures/mutate-create.json [hunk 1]
```diff
@@ -31,6 +34,9 @@
...
 {"code":"ATLAS","make":"GIN","name":"Atlas","class":"EN-B","v_min":null,"three_liner":null,"size":null,"available":null,"certified":null, "color":null,"spec":null},
-{"code":"SPRINT-EVO","make":"GIN","name":"Sprint Evo","class":"EN-B","point":"POINT(11.574601 48.148484)","multipolygon":"MULTIPOLYGON(((11.578791 48.144026,11.582921 48.143088,11.582181 48.141649,11.578136 48.142515,11.578791 48.144026)))"}
+{"code":"SPRINT-EVO","make":"GIN","name":"Sprint Evo","class":"EN-B","point":"POINT(11.574601 48.148484)","multipolygon":"MULTIPOLYGON(((11.578791 48.144026,11.582921 48.143088,11.582181 48.141649,11.578136 48.142515,11.578791 48.144026)))"},
+{"code":"FOOBAR","make":"YANKEE","name":"Yankee","class":"EN-B","line":"LINESTRING(0.123456 0.123456, 0.123456 10.123456, 10.123456 10.123456)"},
+{"code":"FOOBAR","make":"HOTEL","name":"Hotel","class":"EN-B","multipoint":"MULTIPOINT(0.123456 0.123456, 0.123456 10.123456)"},
+{"code":"FOOBAR","make":"FOXTROT","name":"Foxtrot","class":"EN-B","polygon":"POLYGON((0.123456 0.123456, 0.123456 10.123456, 10.123456 10.123456, 0.123456 0.123456))"}
```
Comment (urmilan) [inline soql-server-pg/src/test/resources/fixtures/mutate-create.json]: Wilco fan? :smiley:

Comment (rozap) [inline soql-server-pg/src/test/resources/fixtures/mutate-create.json]: mebbe :walking:

**Comment by urmilan** [comment]
:shipit:

**Comment by chitang** [comment]
The column "code" was meant to be unique.

**Pull request merged**